05-Time-Series-Raster.Rmd

```diff
@@ -92,102 +93,119 @@ This lesson is a part of a series of raster data in R lessons:
 * [Lesson 05 - Raster Time Series Data in R]({{ site.baseurl}}/R/Raster-Times-Series-Data-In-R/)
 * [Lesson 06 - Plot Raster Time Series Data in R Using RasterVis and LevelPlot]({{ site.baseurl}}/R/Plot-Raster-Times-Series-Data-In-R/)
 * [Lesson 07- Extract NDVI Summary Values from a Raster Time Series]({{ site.baseurl}}/R/Extract-NDVI-From-Rasters-In-R/)
-</div>
 
-In this lesson, we will use the `raster`, `rgdal` and `rasterVis` libraries.
+###Additional Resources
+* <a href="http://cran.r-project.org/web/packages/raster/raster.pdf" target="_blank">
+Read more about the `raster` package in `R`.</a>
 
-```{r load-libraries }
+</div>
 
-library(raster)
-library(rgdal)
-library(rasterVis)
+#About Raster Time Series Data
+Raster Data Sets can be made up of multiple files or bands.  These could be
+different specra taken at the same time and location that together offer a
+more complete image of the location. Or these could be single bands taken at
+different times of the same location.  Time series data gives us the ability to 
+measure and monitor changes through time.  
 
```

Comment:
#About Raster Time Series Data
Raster data can be in a single band or data layer format. Or it can contains multiple datasets or bands. 
If the data are images, each band may represent reflectance for a different wavelength (color) or set of wavelengths - for example red, green and blue. Raster data may also contains layers that represent data collected for the same area and of the same resolution. 

**the concept of wavelengths could be confusing. see what you think about this text. I think we explain this in the video and could extract graphics from it easily.